i18n/de.toml

```diff
@@ -92,7 +92,7 @@ about = "Über"
 blog = "Blog"
 docs = "Dokumentation"
 github = "GitHub"
-login = "Fahrzeug Logins"
+login = "Fahrzeug-Logins"
```

Comment:
Warum nicht ein Wort?

Reply:
Is this used in practice at all? What vehicles use this??

Reply:
If it is not utilised anymore - perhaps we could simply remove it ?

Reply:
Its used for the Mercedes Integration.

Reply:
@andig

Siehe z. B.:
- https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich#D26
- https://www.duden.de/sprachwissen/rechtschreibregeln/bindestrich#D22

Reply:
Ist in Weblate angepasst.